.github/workflows/e2e-matrix.yaml

```diff
@@ -33,7 +33,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        suite: [Nonbehavioral, Utilization, GPU, Drift, Integration]
+        suite: [Nonbehavioral, Utilization, GPU, Drift, Integration, NodeClaim]
```

Comment:
@charliedmcb is there somewhere else new that I also should add `NodeClaim` to? I noticed that suite didn't get run from your /test comment.

Reply:
I'm assuming it didn't go because you hadn't merged yet, but I'm not sure

Running the suites again

Reply:
Possibly because it's now using the e2e-matrix.yaml file from the main branch instead of this one.

Reply:
![image](https://github.com/Azure/karpenter/assets/30397998/835b60f4-1a10-4b15-8ec9-41653cad2596)
for reference

Reply:
Yea, I think it is using the mainline. Manual trigger on your branch should allow for testing for now, but will look at improving the workflows for new tests as that's not great

Reply:
I think it was unhappy with running the matrix directly: https://github.com/Azure/karpenter/actions/runs/6817738727